provider/resources.go

```diff
@@ -54,9 +54,7 @@ func Provider() tfbridge.ProviderInfo {
 		P:                 p,
 		Name:              mainPkg,
 		Version:           version.Version,
-		Publisher:         caser.String(publisher),
```

Comment:
@omercnet can you restore the value for `Publisher` to be `Pulumiverse`? We only use `Pulumi Corp.` for the packages maintained in the `pulumi` github organization.

Reply:
done